Indexers in c# (#3776)
section-engineering-education/engineering-education: content/authors/ezra-mukono/index.md

> ---
> title: Ezra Mukono
> type: engineering-education/author
> ---
> Ezra Mukono is a computer science  student. He is much more interested in application development, software development.
>  
>  He also uses his leisure time doing some other sort of researches to improve the already got knwoledge

Reviewer: ```suggestion
---
title: Ezra Mukono
type: authors
---
Ezra Mukono is a computer science student. He is much more interested in application development, software development. He uses his leisure time doing other researches to improve on his knowledge.

```

> ---
> title: Ezra Mukono
> type: engineering-education/author
> ---
> Ezra Mukono is a computer science  student. He is much more interested in application development and software development.
>  
>  He also uses his leisure time doing some other sort of researches to improve the already got knwoledge

Reviewer: ```suggestion
---
title: Ezra Mukono
type: authors
---
Ezra Mukono is a computer science student. He is much more interested in application development and software development, and uses his leisure time doing other researches to improve on his knowledge.
```

Author: @dentonya  What exactly am I supposed to correct

Reviewer: On the suggested changes..just click commit and the changes will be done @mukono10

Reviewer: ![Screenshot_2021-09-15-13-23-26-40_320a9a695de7cdce83ed5281148d6f19.jpg](https://user-images.githubusercontent.com/77575569/133416788-d7d8ca4c-921d-47fd-ad9d-4aa78f907869.jpg)
 Just click the commit button and the changes will be updated...when you're done let me know so that we may continue to next stage 
@mukono10
